experimental/veggieseasons/lib/widgets/trivia.dart

```diff
@@ -150,21 +148,15 @@ class _TriviaViewState extends State<TriviaView> with RestorationMixin {
               ),
               Padding(
                 padding: const EdgeInsets.symmetric(horizontal: 16),
-                child: Text(
-                  ' of ',
-                  style: Styles.triviaFinishedText(themeData),
-                ),
+                child: Text(' of ', style: themeData.textTheme.textStyle),
```

Comment:
In testing your change, I discovered a pre-existing issue on line 146. Would you mind changing it from:

```
'$score',
```

to:

```
'${score.value}',
```